docs/docsite/rst/dev_guide/developing_module_utilities.rst

```diff
@@ -1,18 +1,16 @@
 .. _appendix_module_utilities:
 
-**************************
 Appendix: Module Utilities
-**************************
+``````````````````````````
 
-Ansible provides a number of module utilities that provide helper functions that you can use when developing your own modules. The ``basic.py`` module utility provides the main entry point for accessing the Ansible library, and all Python Ansible modules must, at minimum, import ``AnsibleModule``::
+Ansible provides a number of module utilities that provide helper functions that you can use when developing your own modules. The `basic.py` module utility provides the main entry point for accessing the Ansible library, and all Ansible modules must, at minimum, import from basic.py::
 
-  from ansible.module_utils.basic import AnsibleModule
+  from ansible.module_utils.basic import *
 
 
-The following is a list of ``module_utils`` files and a general description. The module utility source code lives in the ``./lib/ansible/module_utils`` directory under your main Ansible path - for more details on any specific module utility, please see the source code.
-
-.. include:: shared_snippets/licensing.txt
+The following is a list of module_utils files and a general description. The module utility source code lives in the `./lib/module_utils` directory under your main Ansible path - for more details on any specific module utility, please see the source code.
 
+- alicloud_ecs.py - Definitions and utilities for modules working with Alibaba Cloud ECS.
```

Comment:
This new line should be the only change to the `docs/docsite/rst/dev_guide/developing_module_utilities` file. Please remove/undo the other changes.